testnets/sagatestnet/chain.json

```diff
@@ -0,0 +1,135 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "sagatestnet",
+  "status": "live",
+  "network_type": "testnet",
+  "pretty_name": "Saga Testnet",
+  "chain_id": "ssc-testnet-1",
+  "bech32_prefix": "saga",
+  "daemon_name": "sscd",
+  "node_home": "$HOME/.ssc",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "utsaga",
+        "fixed_min_gas_price": 0,
+        "low_gas_price": 0.0025,
+        "average_gas_price": 0.025,
+        "high_gas_price": 0.04
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "utsaga"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/sagaxyz/ssc",
+    "recommended_version": "v0.1.0",
```

Comment:
v0.1.3

Reply:
Thanks @lukitsbrian. See updated changes.

Reply:
I think a few other fields in the json need to be updated as well, otherwise looks good!